### ISSUE TITLE
Manually merge max category clean up PRs

### ISSUE THREAD

**Issue by maniarathi** [chanzuckerberg/cellxgene#2395]
These following two PRs will not cleanly merge into the TS cleanup PR. Please manually merge these changes once the initial TS cleanup is complete.

https://github.com/chanzuckerberg/cellxgene/pull/2347
https://github.com/chanzuckerberg/cellxgene/pull/2354

**Comment by bkmartinjr** [comment]
Resolved by #2402
